x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -125,41 +118,32 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
     license_level,
   };
 
-  const {
-    services: { share },
-  } = useMlKibana();
-
   useEffect(
-    function updateDeploymentState() {
+    function updateModelItems() {
       (async function () {
-        const { nodes, ...deploymentStats } = stats.deployment_stats ?? {};
-
-        if (!isPopulatedObject(deploymentStats)) return;
+        const deploymentStats = stats.deployment_stats;
 
-        const result = formatToListItems(deploymentStats)!;
+        if (!deploymentStats) return;
 
-        const items: EuiListGroupItemProps[] = await Promise.all(
-          nodes!.map(async (v) => {
-            const nodeObject = Object.values(v.node)[0];
-            const href = await mlLocator!.getUrl({
-              page: ML_PAGES.TRAINED_MODELS_NODES,
-              pageState: {
-                nodeId: nodeObject.name,
-              },
-            });
-            return {
-              label: nodeObject.name,
-              href,
-            };
-          })
-        );
-
-        result.push({
-          title: 'nodes',
-          description: <EuiListGroup size={'s'} gutterSize={'s'} listItems={items} />,
+        const items: AllocatedModel[] = deploymentStats.nodes.map((n) => {
+          const nodeName = Object.values(n.node)[0].name;
+          return {
+            ...deploymentStats,
+            node: {
+              ...pick(n, [
+                'average_inference_time_ms',
+                'inference_count',
+                'routing_state',
+                'last_access',
+                'number_of_pending_requests',
+                'start_time',
+              ]),
+              name: nodeName,
+            } as AllocatedModel['node'],
+          };
         });
 
-        setDeploymentStats(result);
+        setModelItems(items);
```

Comment:
The deployment stats panel needs bottom padding I think:

![image](https://user-images.githubusercontent.com/7405507/141969417-eabcd406-4948-43d2-884d-bf1b09333cf6.png)

Reply:
Fixed in 2b358860a4bd6a821169a561d2d85a73badb8093